apps/tests_files/test/test.c

```diff
@@ -0,0 +1,6 @@
+int main() {
```

Comment:
Do we need this test?

Reply:
yes its the most simple test to make sure everything is fine